src/main/java/com/karankumar/bookproject/ui/settings/SettingsView.java

```diff
@@ -97,7 +104,7 @@
 
         configureExportBooksAnchor();
 
-        VerticalLayout verticalLayout = new VerticalLayout(horizontalLayout, clearShelvesButton, exportBooksAnchor);
+        VerticalLayout verticalLayout = new VerticalLayout(appearanceHeading, horizontalLayout, lineBreak, myBooksHeading, clearShelvesButton, exportBooksAnchor);
```

Comment:
As this violates our 100 character column limit, could you please refactor this to something like the following:

```java
        VerticalLayout verticalLayout = new VerticalLayout(
                appearanceHeading,
                horizontalLayout,
                lineBreak,
                myBooksHeading,
                clearShelvesButton,
                exportBooksAnchor
        );
```

Reply:
Done